### ISSUE TITLE
after about 1000 training steps, training loss continues to rise

### ISSUE THREAD

**Issue by Alex417ii** [OpenRobotLab/AnySplat#88]
When training the AnySplat model on the DL3DV_10k_samples dataset, I kept most configurations unchanged except for setting num_context_views: 8, max_img_per_gpu: 4, and the data loading thread number to 1. However, after about 1000 training steps, I encountered the message: "Skipping batch with high loss (0.249768) at step 1001 on Rank 25". Subsequently, the training loss continues to rise, even reaching around 1.5. Could you explain what might be causing this and suggest any possible solutions? I look forward to your reply. Thank you!

**Comment by myc634** [comment]
how many gpu you are using? what is the global batch size? anysplat training relies on high batch size, and is unstable under small batch size, I suggest following the original setting to reproduce.
